### PR TITLE
insert newline after triple backslashes if not present

### PR DESCRIPTION
**Pull request by dbarrosop**
It's quite often that users post code in slack like this:

    ```if blah:
        return True
    ```

This simple patch should make this more readable by inserting a newline after the triple backslashes if not present already:

    ```
    if blah:
        return True
    ```

### CODE REVIEW

File: wee_slack.py [hunk 1]
```diff
@@ -1407,6 +1407,7 @@ def destroy_buffer(self, update_remote):
             self.eventrouter.receive(s)
 
     def buffer_prnt(self, nick, text, timestamp=str(time.time()), tagset=None, tag_nick=None, **kwargs):
+        text = re.sub(r"```(?!\n)", "```\n", text)
```
Comment (trygveaa) [inline wee_slack.py]: When the message changes, the message will be rendered again, which will overwrite this. To make this persist through message edits, you need to place this in the function called `render` instead.

This adds a newline at the end of the message if the message ends with ```. This should not happen.

**Comment by dbarrosop** [comment]
Will look at it in a few days, thanks!

File: wee_slack.py (hunk 1 above)
Comment (dbarrosop) [reply wee_slack.py]: I think both comments are now addressed

**Comment by trygveaa** [comment]
Now, it adds a blank line before the opening \`\`\`. It also removes the first character after \`\`\` if there is no newline between them.

**Pull request closed without merging**